pom.xml

```diff
@@ -64,6 +64,7 @@
     </scm>
     <properties>
         <janusgraph.compatible.versions />
```

Comment:
Doesn't this field have to list the current version, i.e., `0.1.0-SNAPSHOT`, to be compatible with JanusGraph itself, at the same version? Otherwise, if you run JanusGraph the first time to create a graph, then stop JanusGraph, and start it up again, it will not be compatible with itself.

It doesn't appear that self-version is automatically compatible with itself from looking at the code, unless I'm missing it.

Have you tested this change? Does it fix the compatibility issues with Titan, and does it work with JanusGraph-created graphs as well?